codice/ddf
# Upgrade to Karaf 4.3.0

#### What does this PR do?
Upgrades Apache Karaf to 4.3.0
#### Who is reviewing it? 
@jMoneee 
@frnkshin 
@pklinef 
@shaundmorris 
@kentmorrissey 
@zta6 

#### Select relevant component teams: 


#### Ask 2 committers to review/merge the PR and tag them here.


#### How should this be tested?
Full build to ensure all tests pass
Start DDF, install profile, verify no regression observed when testing basic functionality

#### Any background context you want to provide?
Karaf 4.3.0 introduced the Karaf bom which manages dependency versions so there were a bunch of those removed in this PR.
#### What are the relevant tickets?
Fixes: #____

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
